rpm/product.spec

```diff
@@ -203,17 +208,26 @@ then
   # Run genesisInstall
   echo "Running Genesis Install script"
   runuser -l "$genesis_user" -c 'genesisInstall'
-
+  install_error_code=$(echo $?)
```

Comment:
I think this would be simpler as `install_error_code=$?` but it equates to the same thing